bindings/SofaRuntime/src/SofaPython3/SofaRuntime/Module_SofaRuntime.cpp

```diff
@@ -81,7 +91,25 @@ class SofaInitializer
     }
 };
 
+static std::vector<std::string>  getCategories(std::string className)
+{
+    std::vector<std::string> categories;
+    ObjectFactory* factory = ObjectFactory::getInstance();
 
+    if (factory->hasCreator(className))
+    {
+        ObjectFactory::ClassEntry entry = factory->getEntry(className);
+        ObjectFactory::CreatorMap::iterator it2 = entry.creatorMap.begin();
+
+        if( it2 != entry.creatorMap.end())
+        {
+                ObjectFactory::Creator::SPtr c = it2->second;
```

Comment:
I suggest to remove the `2` suffix
```suggestion
        ObjectFactory::CreatorMap::iterator it = entry.creatorMap.begin();

        if( it != entry.creatorMap.end())
        {
                ObjectFactory::Creator::SPtr c = it->second;
```